### PR TITLE
Component Governance fix round 3

### PR DESCRIPTION
**Pull request by jchen351**

### CODE REVIEW

File: tools/ci_build/github/azure-pipelines/templates/c-api-linux-cpu.yml [hunk 1]
```diff
@@ -33,14 +33,18 @@ parameters:
...
+  default: false
+
 jobs:
 - job: Linux_C_API_Packaging_CPU_${{parameters.OnnxruntimeArch}}
 
   workspace:
     clean: all
   variables:
   - name: skipComponentGovernanceDetection
-    value: ${{ in('${{parameters.OnnxruntimeNodejsBindingArch}}', 'arm64', 'aarch64') }}
```
Comment (jchen351) [inline tools/ci_build/github/azure-pipelines/templates/c-api-linux-cpu.yml:43]: Somehow this is True when arch is x64 and False when the arch is arm64

Comment (snnn) [reply tools/ci_build/github/azure-pipelines/templates/c-api-linux-cpu.yml:43]: What if you totally remove this condition, what would happen?

Comment (jchen351) [reply tools/ci_build/github/azure-pipelines/templates/c-api-linux-cpu.yml:43]: If we remove the ${{ in('${{parameters.OnnxruntimeNodejsBindingArch}}', 'arm64', 'aarch64') }}, we need to it to either True or False. Value cannot be unset.

Comment (snnn) [reply tools/ci_build/github/azure-pipelines/templates/c-api-linux-cpu.yml:43]: I mean, what blocks us running component governance on ARM64? Can we fix that?

Comment (jchen351) [reply tools/ci_build/github/azure-pipelines/templates/c-api-linux-cpu.yml:43]: I thought it wasn't supposed on arm64?

Comment (jchen351) [reply tools/ci_build/github/azure-pipelines/templates/c-api-linux-cpu.yml:43]: Let me remove it and see

**Review by snnn — APPROVED** [review]

**Pull request merged**